cg/meta/orders/rnafusion_submitter.py

```diff
@@ -1,5 +1,19 @@
+from typing import List
+
+from cg.exc import OrderError
 from cg.meta.orders.case_submitter import CaseSubmitter
+from cg.models.orders.order import OrderIn
+from cg.models.orders.samples import Of1508Sample
 
 
 class RnafusionSubmitter(CaseSubmitter):
-    pass
+    def validate_order(self, order: OrderIn) -> None:
+        """Validates that the order is correct."""
```

Comment:
Maybe the doc string can be omitted here since the function name contains the same information